src/inject/dynamic-theme/css-rules.ts

```diff
@@ -58,22 +58,25 @@ export function iterateCSSDeclarations(style: CSSStyleDeclaration, iterate: (pro
         }
         iterate(property, value);
     });
-    if (isSafari && style.cssText.includes('var(')) {
-        // Safari doesn't show shorthand properties' values
-        shorthandVarDepPropRegexps.forEach(([prop, regexp]) => {
-            const match = style.cssText.match(regexp);
-            if (match && match[1]) {
-                const val = match[1].trim();
-                iterate(prop, val);
-            }
-        });
-    } else {
-        shorthandVarDependantProperties.forEach((prop) => {
-            const val = style.getPropertyValue(prop);
-            if (val && val.includes('var(')) {
-                iterate(prop, val);
-            }
-        });
+    const cssText = style.cssText;
+    if (cssText.includes('var(')) {
```

Comment:
First of all there was previously a bug I forgot to fix. It should be `if (isSafari) { if(style.cssText.includes...) {... } } else { ... }`.
Secondly I'm not sure if `cssText.includes('var(')` should be done for other browsers. It's hard to find out if checking every style for "var(" is faster than running `style.getPropertyValue(prop)` for each style several times. Worth making some benchmark, for Firefox too.

Reply:
Firefox seems to lack behind - Chromium can in some cases have better performance with the `includes`. But not worth it, not sure how the test cases are against actually world code like gmail. But then we are speaking of thousands of such styles. Which all vary

Reply:
https://jsben.ch/3jFp6